GNUmakefile

```diff
@@ -0,0 +1,14 @@
+include $(GNUSTEP_MAKEFILES)/common.make
+
+BUNDLE_NAME = MIPSCPU
+MIPSCPU_OBJC_FILES = \
+	MIPSCPU/MIPSCPU.m \
+	MIPSCPU/MIPSCtx.m
+
+#NOWARN=-Wno-format -Wno-return-type -Wno-unused-value -Wno-unused-variable -Wno-self-assign
+MIPSCPU_OBJCFLAGS=$(NOWARN) \
+	-I${HOME}/GNUstep/Library/ApplicationSupport/Hopper/HopperSDK/include \
```

Comment:
The reason I went with my path to the SDK is because I have an [installer](https://github.com/ckuethe/HopperSDK-Linux) that matches the SDK version to the version of Hopper that is installed.